packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,710 @@
+---
+title: Factor Profiles
+category: management
+---
+
+# Factor Profiles API
+
+The Okta Factor Profiles API enables an Administrator to configure which Factor Profiles are available to use for multi-factor authentication.
+
+> **Note:** In the Admin Dashboard a "Factor Profile" is referred to as an "Authenticator".
+
+The Factor Profiles API supports the following operations:
+
+* Create a Factor Profile
+* Update a Factor Profile
+* Get a Factor Profile
+* Get all Factor Profiles by Factor Name
```

Comment:
General question, don't we refer to getting multiple entries as `List`? https://developer.okta.com/docs/reference/api/authorization-servers/#list-authorization-servers for example.